README.pt-br.md

```diff
@@ -602,11 +620,207 @@ ObxValue(
 ),
 ```
 
-## Explicação em vídeo sobre Outras Features do GetX
-
+### Explicação em vídeo sobre Outras Features do GetX
 
 Amateur Coder fez um vídeo incrível sobre utils, storage, bindings e outras features! Link: [GetX Other Features](https://youtu.be/ttQtlX_Q0eU)
 
+
+## Dicas Úteis
+
+`.obs`ervables (também conhecidos como _Rx_ Types) possuem uma grande variedade de métodos e operadores internos.
+
+> É muito comum acreditar que uma propriedade com `.obs` **É** o valor real... mas não se engane!
+> Evitamos a declaração de tipo da variável, porque o compilador do Dart é inteligente o suficiente e o código
+> parece mais limpo, mas:
+
+```dart
+var message = 'Hello world'.obs;
+print( 'Message "$message" é do tipo ${message.runtimeType}');
+```
+
+Mesmo que `message` _imprima_ o valor da string, seu tipo é **RxString**!
+
+Então, você não pode fazer `message.substring( 0, 4 )`.
+Você tem que acessar o `valor` real dentro do _observable_:
+A "maneira" mais usada é utilizando `.value`, mas, você sabia que também pode usar:
+
+```dart
+final name = 'GetX'.obs;
+// apenas "atualiza" o stream, se o valor for diferente do atual.
+name.value = 'Hey';
+
+// Todas as propriedades Rx são "chamáveis" e retorna o novo valor.
+// mas esta abordagem não aceita `null`, a UI não será reconstruída
+name('Hello');
+
+// é como um getter, imprime 'Hello'
+name() ;
+
+/// números:
+
+final count = 0.obs;
+
+// Você pode usar todas as operações não mutáveis ​​das primitivas de num! 
+count + 1;
+
+// Cuidado! isso só é válido se `count` não for final, mas var
+count += 1;
+
+// Você também pode comparar com os valores:
+count > 2;
+
+/// booleans:
+
+final flag = false.obs;
+
+// mude o valor entre true/false
+flag.toggle();
+
+
+/// todos os tipos:
+
+// Defina `value` como null.
+flag.nil();
+
+// Todas as operações toString() e toJson() são passada para `value`
+print( count ); // chama `toString()` de RxInt
+
+final abc = [0,1,2].obs;
+// Converte o valor em um Array json, imprime RxList
+// Json é suportado por todos os Rx types!
+print('json: ${jsonEncode(abc)}, type: ${abc.runtimeType}');
+
+// RxMap, RxList e RxSet são Rx types especiais, que estendem seus tipos nativos.
+// mas você pode trabalhar com uma lista como uma lista normal, embora seja reativa!
+abc.add(12); // Coloca 12 na lista, e ATUALIZA o stream.
+abc[3]; // como uma lista lê o índice 3.
+
+// a igualdade funciona com o Rx e o valor, mas hashCode é sempre obtido do valor 
```

Comment:
esse aqui parece certo mas eu não sei o que é hasCode
oq vc quis dizer com "o hashCode é sempre obtido do valor"?

Reply:
Na documentação em inglês está escrito assim "// equality works with the Rx and the value, but hashCode is always taken from the value", 
O correto seria assim, o que acha?
`// a igualdade funciona com o Rx e o value do observável, mas o hashCode é sempre obtido do value`

Na documentação do Dart (https://api.dart.dev/stable/2.9.3/dart-core/Object/hashCode.html) diz que o hashcode é um inteiro único que representa o estado do objeto que afeta as comparações.

Reply:
> O correto seria assim, o que acha?
// a igualdade funciona com o Rx e o value do observável, mas o hashCode é sempre obtido do value

por mim tá excelente 😄 
não entendi muito bem mas meu entendimento tá fora do escopo desse PR